pkg/sources/github/events.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package github
+
+const (
+	// Event Names
+	PullrequestEvent = "dev.knative.github.pullrequest"
+	UnsupportedEvent = "dev.knative.github.unsupported"
+)
+
+var CloudEventType = map[string]string{
+	"pull_request": PullrequestEvent,
+	"":             UnsupportedEvent,
+}
+
+var GithubEventType = map[string]string{
```

Comment:
Is this just the reverse of `CloudEventType`?

Reply:
yes, is there a better way?

Reply:
Build it automatically, like:

```golang
var GithubEventType map[string]string
for k, v := range(CloudEventType) {
  GitHubEventType[v] = k
}
```